recipes/datreant.data/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.6.0" %}
+{% set sha256 = "a124c0598ac55b71de43447a2398b1508452e07ea7d7369ee70428a1de9f977f"%}
+
+package:
+  name: datreant.data
+  version: {{ version }}
+
+source:
+  fn: datreant.data-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/d/datreant.data/datreant.data-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - datreant
+    - datreant.core
+    - numpy
+    - pandas
+    - six
+    - pytables
+    - h5py
+
+test:
+  imports:
+    - datreant.data
+  requires:
+    - pytest
+
+about:
+  home: http://datreant.org/
+  license: BSD 3-Clause
```

Comment:
In the next official release I can also add the license but currently this is not possible.